torch/csrc/jit/codegen/cuda/tensor_view.cpp

```diff
@@ -290,40 +291,117 @@ IterDomain* TensorView::axis(int pos) const {
   return domain()->axis(pos);
 }
 
-void TensorView::setComputeAt(unsigned int pos, bool decrease) {
+void TensorView::inlineAt(int64_t pos, bool best_effort) {
   TORCH_INTERNAL_ASSERT(
       !container()->isA<kir::Kernel>(),
       "Function invalid for kernel container.");
-  if (pos <= compute_at_pos_ && !decrease) {
-    return;
+
+  std::unique_ptr<InlineBatchingGuard> maybe_guard;
+  if (InlineBatchingGuard::getCurrentGuard() == nullptr) {
+    maybe_guard = std::make_unique<InlineBatchingGuard>(fusion());
+  }
+
+  TORCH_INTERNAL_ASSERT(
+      fusion() == InlineBatchingGuard::getCurrentGuard()->getFusion(),
```

Comment:
This would segfault when `getCurrentGuard()` is nullptr.

Reply:
This is removed now. Then entire `InlineBatchingGuard` is removed